syntax = "proto3";
import "DeviceInfoStructure.proto";

message DeviceInfoResponse {
	DeviceInfoStructure deviceInfo = 1;
}
